### DISCUSSION TITLE
[需要更新]星露谷物语

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#291] [投递和请求游戏]
游戏名称：星露谷物语
游戏名称(英文)：Stardew Valley
游戏版本：1.5.6（主站为1.4.5）
下载地址 :（如果有的话）：https://cloud.189.cn/web/share?code=ArUfqmuqqquq （访问码：qc1q）
仅64位版本

**Comment by hyh19962008** [comment]
已更新
